mfthomps/CyberCIEGE-port-to-Unity3d
# issue/307-nav-mesh-for-all-offices

Closes #307 

This should prevent most warnings/errors about NavMeshAgents.

To verify
* run a few of the scenarios that have offsite offices and ensure nothing weird is happening.

This adds in nav meshes for the offsite offices (including the WWW, in case one wants to walk through the internet).
